paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java

```diff
@@ -202,4 +205,37 @@ static MySqlSource<String> buildMySqlSource(Configuration mySqlConfig) {
                 new JsonDebeziumDeserializationSchema(true, customConverterConfigs);
         return sourceBuilder.deserializer(schema).includeSchemaChanges(true).build();
     }
+
+    private static void validateMySqlConfig(Configuration mySqlConfig) {
```

Comment:
Can we invoke `org.apache.flink.table.factories.FactoryUtil.validateFactoryOptions`?